apps/oc_chef_wm/src/oc_chef_object_db.erl

```diff
@@ -117,6 +117,12 @@ maybe_delete_authz_id_or_error(1, Object, _RequestorId)  when is_record(Object,
                                                               is_record(Object, oc_chef_org_user_invite) ->
     %% These objects have no authz id
     ok;
+maybe_delete_authz_id_or_error({ok, 1}, #oc_chef_cookbook_artifact_version{authz_id = AuthzId}, RequestorId) ->
+    case oc_chef_cookbook_artifact:exists_by_authz_id(AuthzId) of
```

Comment:
It seems to me that's actually needed. See the same thing for cookbook_versions just below that.